packages/react-core/src/components/Label/__tests__/Label.test.tsx

```diff
@@ -84,9 +84,18 @@ describe('Label', () => {
     expect(asFragment()).toMatchSnapshot();
   });
 
+  test('label with isTruncated turned off', () => {
+    const { asFragment } = render(
+      <Label className="class-1" id="label-1" data-label-name="something" isTruncated={false}>
+        Something that is pretty long and won't get truncated
+      </Label>
+    );
+    expect(asFragment()).toMatchSnapshot();
+  });
+
   test('label with truncation', () => {
     const { asFragment } = render(
-      <Label isTruncated>Something very very very very very long that should be truncated</Label>
+      <Label>Something very very very very very long that should be truncated</Label>
```

Comment:
This needs a val for `textMaxWidth` to trigger the truncation.

```suggestion
      <Label textMaxWidth="16ch">Something very very very very very long that should be truncated</Label>
```